static/js/main.js

```diff
@@ -0,0 +1,49 @@
+const toggleClass =  (el, cls) => {
+    if (Array.isArray(cls)) {
+        cls.map((cl) => {
+            el.classList.toggle(cl);
+        });
+    } else {
+        el.classList.toggle(cls);
+    }
+};
+
+function DarkModeIndex() {
+    var footer = document.getElementById('footer')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zlataovce/plshelp/pullRequest?prid=7177914)
```suggestion
    var footer = document.getElementById("footer")
```